doc/postgres.txt

```diff
@@ -93,3 +93,32 @@
    sudo -u postgres psql -d testreport -c "select * from lkft_reportjob;"
    sudo -u postgres psql -d testreport -c "select count(*) from lkft_testcase;"
    sudo -u postgres psql -d testreport -c "SELECT * FROM lkft_testcase WHERE ( job_id = '2476852' AND lava_nick = 'production' AND suite LIKE '%_scimark' AND name = 'scimark-Monte-Carlo');"
+
+5. instructions on moving the database files to another place
+    $ sudo systemctl stop postgresql@13-main
+    $ sudo rsync -av /var/lib/postgresql/13/ postgres_13/
+    ## when run with the postgres user
+    $ rsync -av /var/lib/postgresql/13/ postgres_13/
+
+    $ vi /etc/postgresql/13/main/postgresql.conf
```

Comment:
I wouldn't touch this. Just symlink from /srv/postgres to /var/lib or mount the data volume in the /var/lib location in fstab